asterius/rts/rts.gc.mjs

```diff
@@ -657,38 +743,54 @@ export class GC {
     }
   }
 
+  /**
+   * Allocates a new nursery and stores its address in the appropriate
+   * field of the StgRegTable of the main capability.
+   */
   updateNursery() {
+    // Note: the 'rHpAlloc' field of the 'StgRegTable' C struct contains 
+    // the number of bytes being allocated in the heap. Here, this field is
+    // read in the hp_alloc variable and used as the size of the newly 
```

Comment:
"As the size" isn't quite right. E.g. if we're allocating a closure of 8 bytes, and we ran out of nursery space, but we can't have a nursery with exactly 8 bytes' size. So "used to determine the size" shall be more precise.